modules/core/src/main/java/org/apache/ignite/internal/processors/cache/persistence/snapshot/IgniteSnapshotManager.java

```diff
@@ -2299,6 +2302,7 @@ public IgniteFutureImpl<Void> createSnapshot(
             }
 
             List<String> grps = (dump ? cctx.cache().cacheGroupDescriptors().values() : cctx.cache().persistentGroups()).stream()
+                .filter(g -> cacheGroupNames == null || cacheGroupNames.contains(g.cacheOrGroupName()))
```

Comment:
What if `cacheGroupNames` param contains non-existent cache group. Should we stop snapshot process? Or at least write a warning.

Reply:
Warning added. Thanks.